Recipes/SBPLAT/apps_copyFromPublicApps.ipynb

```diff
@@ -98,11 +98,289 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
So you are choosing to show the cell outputs here as a helpful list for users. I think this is ok, but wanted to make sure this is a conscious choice.